src/content/blog/2024_zne_catalyst.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Just in time for just-in-time error mitigation: Mitiq meets Catalyst"
+author: Alessandro Cosentino
+day: 18
+month: 11
+year: 2024
+tags:
+- Mitiq
+- Partnership
+---
+
+This blog post is a summary of a partnership between Unitary Fund and 
+[Xanadu](https://www.xanadu.ai/) to integrate quantum error mitigation techniques
+from Unitary Fund's Mitiq into Xanadu's PennyLane Catalyst compiler.
```

Comment:
@cosenal I wonder if it is worth linking Mitiq and Catalyst websites here, on their first mention? But this is optional, as they are linked right below 🤔

Reply:
I thought about it, but I prefer below where they are introduced. This paragraph is more like a tl;dr and I don't want to drive readers out of it immediately by clicking on too many links.